tests/tblgen_to_py/test_tblgen.py

```diff
@@ -11,5 +11,4 @@ def test_run_tblgen_to_py():
     with open("tests/tblgen_to_py/test.py") as f:
         expected = f.read()
 
-        assert len(out_str.strip()) == len(expected.strip())
```

Comment:
Seems unrelated but guess it got annoying?

Reply:
Oh yeah it was actually making the test harder to debug, pytest shows a nice diff for string comparisons, and not for `len`s. I meant to put it in a different PR, but if you're ok to merge it then we could just do it in this one.

Reply:
I reverted it here, and will open a separate PR for this.